website/static/js/share/index.js

```diff
@@ -87,6 +92,14 @@ ShareApp.controller = function() {
 
     });
 
+    m.request({
+        method: 'GET',
+        url: '/api/v1/share/documents/' // TODO where will the postgres API live??
+    }).then(function(data) {
+        self.vm.rawNormedLoaded(true);
+    }, function(err) {
+        // blank, but needed to handle missing rawNormed API
```

Comment:
Is an error expected here? If not, then the error should be handled (log to Sentry, show message, etc.)